fetch/api/basic/accept-header.any.js

```diff
@@ -1,4 +1,5 @@
 // META: script=../resources/utils.js
+// META: global=window,worker
```

Comment:
I suggest we put global before script. (title before global seems reasonable though.)